config/test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "port": "3061",
```

Comment:
Modified this so that I could run my dev server AND run the test suite at the same time, otherwise there is a port collision.